client/src/App.js

```diff
@@ -1,14 +1,40 @@
 import React, { Component } from 'react';
 import Nav from './components/nav/nav';
 import Footer from './components/footer/footer';
+import { Query } from "react-apollo";
+import gql from "graphql-tag";
 
 class App extends Component {
 	render() {
 		return (
 			<div className='App'>
 				<Nav />
 				<Footer />
-			
+
+				<Query
+					query={gql `
+						{
```

Comment:
While this is a great example, it's not particularly useful on app.js If you want to demonstrate it can you put it in a component and maybe import it?